app/views/exchanges/hbx_profiles/_edit_fein.html.erb

```diff
@@ -0,0 +1,39 @@
+<%if @errors_on_save %>
+  <div class='alert alert-error'><a class='close' data-dismiss='alert'>x</a>
+  <% @errors_on_save.each do |error| %>
+    <li><%= error %></li>
+  <% end %>
+  </div>
+<% end %>
+
+<div id=“change_fein”>
+  <div class="bottom-pd">
+    <%= form_for @organization, remote: true, url: update_fein_exchanges_hbx_profiles_path(id: params["id"], employer_actions_id: "employer_actions_#{@organization.employer_profile.id.to_s}"), method: "post", class: "form" do |f|%>
+      <div class="row no-buffer form-wrapper-buffer">
+        <div class="pull-right">
+          <a href="javascript:void(0);" onclick="$('tr.child-row:visible').remove();">
+            <span aria-hidden="true"> <i class="fa fa-times fa-lg"></i> </span>
+          </a>
+        </div>
+        <h3 class="title text-center">Change FEIN</h3>
```

Comment:
Update the text using translations

Reply:
updated text using translations, thanks.